src/main/pages/che-7/end-user-guide/proc_mounting-a-secret-as-a-file-into-a-workspace-container.adoc

```diff
@@ -34,7 +34,7 @@ metadata:
 ...
 ----
 +
-Annotations must indicate the given secret is mounted as a file, provide the mount path, and, optionally, specify the name of the container in which the secret is mounted. If there is no target-container annotation, the secret will be mounted into all user containers of the {prod-short} workspace.
+Annotations must indicate the given secret is mounted as a file, provide the mount path, and, optionally, specify the name of the container in which the secret is mounted. If there is no target-container annotation, the secret will be mounted into all user containers of the {prod-short} workspace, but this is applicable only for the {prod-short} version {prod-prev-ver}.
```

Comment:
I think we should not use versions placeholder here, because it may change in future, but feature is applien in concrete release.

Reply:
I will create new attributes for this since we have to have it CRW docs as well.